docs/bots-guide.md

```diff
@@ -443,6 +443,103 @@ bot_handler.update_message(dict(
   development environment hosted on your computer, use
   `localhost:9991`
 
+### Writing tests for bots
+
+Bots in the [`python-zulip-api`](https://github.com/zulip/python-zulip-api/tree/master/zulip_bots/zulip_bots/bots)
+repository should have unittests. This section will explain the basics of writing a test suite for a bot.
```

Comment:
I would say "Bots, like most software that you want to work, should have unit tests.  In this section, we detail our framework for writing unit tests for bots.  We require that bots in the main `python-zulip-api` repository include a reasonable set of unit tests, so that future developers can easily refactor them."